p/nvv/products.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+module.exports = [
+	// todo: what is `256`?
+	{
+		id: 'express',
+		mode: 'train',
+		bitmasks: [1],
+		name: 'InterCityExpress',
```

Comment:
Can this also be `TGV`, for example?

Reply:
Apparently TGV trains are not shown in the NVV app.